types/react-bootstrap-table/index.d.ts

```diff
@@ -88,19 +229,19 @@ export interface BootstrapTableProps extends Props<BootstrapTable> {
 	 * If a string given, means the value will be presented as the row class.
 	 * If a function given, will pass rowData and rowIndex as params and should return string for presenting class. for examples:
 	 * @example
-	 * 		function trClassFormat(rowData,rowIndex){
-	 * 			return rowIndex%2==0?"tr-odd":"tr-even";  //return a class name.
-	 * 		}
+	 *    function trClassFormat(rowData,rowIndex) {
+	 *      return rowIndex % 2 == 0 ? "tr-odd" : "tr-even"; // return a class name.
+	 *    }
 	 */
-	trClassName?: string | ((rowData: any, rowIndex: number) => string);
+	trClassName?: string | ((rowData: Row<any>, rowIndex: number) => string);
```

Comment:
`Row<any>` just becomes `any`.

Reply:
Suggestion (not required for approval): If you make `BootstrapTableProps` generic as I mention above, the definition for this would be:

```ts
trClassName?: string | (rowData: TRow, rowIndex: number) => string);
```